.devcontainer/devcontainer.json

```diff
@@ -13,18 +13,28 @@
             "name": "Graasp database",
             "driver": "PostgreSQL",
             "previewLimit": 50,
-            "server": "db",
+            "server": "graasp-postgres",
             "port": 5432,
             "database": "docker",
             "username": "docker",
             "password": "docker"
           },
+          {
+            "name": "Graasp Test database",
+            "driver": "PostgreSQL",
+            "previewLimit": 50,
+            "server": "graasp-postgres-test",
+            "port": 5432,
+            "database": "docker-test",
+            "username": "docker-test",
+            "password": "docker-test"
+          },
           {
             "name": "Etherpad database",
             "driver": "PostgreSQL",
             "previewLimit": 50,
-            "server": "db",
-            "port": 5433,
+            "server": "graasp-postgres-etherpad",
+            "port": 5432,
```

Comment:
Shouldn't this be `5433`? I'm not quite sure how the settings work, but you changed the value here but not in `docker-compose`. Or was it a bug?

Reply:
@pyphilia This took me an entire afternoon to understand:
- inside the dev-container the other containers are addressed by their `container-name` and `internal ports`, in this case the etherpad database can be reached using `host=graasp-postgres-etherpad` and `port=5432` the exposed port `5433` is only used when accessing from the host (your terminal on your machine)
- I tested and while the previous configuration was unable to open a connection with the sql-tools extensions in vs-code, with this one it works.